pkg/proxy/iptables/proxier.go

```diff
@@ -1609,11 +1618,29 @@ func (proxier *Proxier) syncProxyRules() {
 
 	// Finally, tail-call to the nodeports chain.  This needs to be after all
 	// other service portal rules.
-	writeLine(proxier.natRules,
-		"-A", string(kubeServicesChain),
-		"-m", "comment", "--comment", `"kubernetes service nodeports; NOTE: this must be the last rule in this chain"`,
-		"-m", "addrtype", "--dst-type", "LOCAL",
-		"-j", string(kubeNodePortsChain))
+	addresses, err := utilproxy.GetNodeAddress(proxier.nodePortAddresses, proxier.networkInterface)
+	if err != nil {
+		glog.Errorf("Failed to get node ip address matching nodeport cidr")
+	} else {
+		for address := range addresses {
+			if utilproxy.IsZeroCIDR(address) {
```

Comment:
If GetNoideAddress() returns both v4 and v6 zero CIDRs, we get 2 rules here?  That seems wrong.

Reply:
Yeah, it's not correct. 

It can be fixed by "break" this loop when `utilproxy.IsZeroCIDR(address)`. PTAL.

Reply:
It's fixed now. PTAL.

Reply:
Please add a commment like:

"TODO(m1093782566,thockin): If/when we have dual-stack support we will want to distinguish v4 from v6 zero-CIDRs."